### DISCUSSION TITLE
请问，如何设置手续费。

### DISCUSSION THREAD

**Discussion by chentianqi-3** [Chia-Network/chia-blockchain#17176] [Support]
请问，如何设置手续费。
我想消除INVALID_FEE_TOO_CLOSE_TO_ZERO报错，想消除提高命中率，想试试。

**Comment by BrandtH22** [comment]
@chentianqi-3 , Please note this has been translated using translation software. 
That error indicates that the fee used is below the minimum effective fee. The minimum effective fee is 5 mojo x the clvm cost of the transaction, anything less will result in that error while there is fee pressure. To help with this we have added a list of estimated minimum effective fees for various standard transaction types here (https://docs.chia.net/coin-set-costs/#estimated-transaction-costs). 

Lastly, we can generally provide more timely and thorough support on our discord server (https://discord.gg/chia)

---

请注意，这是使用翻译软件翻译的。 该错误表明所使用的费用低于最低有效费用。最低有效费用为 5 mojo x 交易的 clvm 成本，在存在费用压力时，任何低于此费用都会导致该错误。为了帮助解决这个问题，我们在这里添加了各种标准交易类型的估计最低有效费用列表（https://docs.chia.net/coin-set-costs/#estimated-transaction-costs）。 最后，我们通常可以在 discord 服务器上提供更及时、更彻底的支持（https://discord.gg/chia）

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
